pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.willowtreeapps</groupId>
     <artifactId>conductor-mobile</artifactId>
-    <version>0.8.0</version>
+    <version>0.8.5</version>
```

Comment:
Good job keeping this on the correct working version. This really helps managing releases!